charts/prometheus-smartctl-exporter/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.11.0
+version: 0.11.1
```

The change should be a minor one:

```suggestion
version: 0.12.0
```